bindings/cpp/src/lib.rs

```diff
@@ -26,6 +26,14 @@ pub struct SecureStorageWaleltInfo {
     pub mnemonic: String,
     pub password: String,
```

Comment:
this was there previously, but I'm just wondering whether the password should be stored (rather than just remembered)... maybe it could be made optional (as in a developer could choose whether the user should always type in the extra password, or store it)?

Reply:
probably no need to change in this PR

Reply:
password is used for address generation.
so if password is different, address also different.

Reply:
file under AndroidSecureStorage, are to compile`SecureStorage.kt`to class file.

which is linked in final android project.

Reply:
> password is used for address generation.
> so if password is different, address also different.

yes, I know, but was thinking whether it should be kept separate, i.e. it's not stored anywhere... HW wallet only keeps the seed, the password is always entered.
anyway, ok to keep it as it is, we can revisit later